cmd_clean/main.go

```diff
@@ -63,9 +63,9 @@ func main() {
 
 	logf.SetLogger(logger.ZapLogger(cf.DefaultConfig.Debug, config.LogLevel))
 
-	status, err := clean.Clean(cf)
+	err := clean.Clean(cf)
 	if err != nil {
-		log.Error(err, "failed to clean nsx resources", "status", status)
+		log.Error(err, "failed to clean nsx resources", "status", err.Error())
 		os.Exit(1)
```

Comment:
Question - was the intention of the status code to also have `os.Exit()` return the code number?

Reply:
we do not intent to use error code as exit code since log message already contains enough information.